cmd/scheduler/app/options/validation.go

```diff
@@ -0,0 +1,31 @@
+package options
+
+import (
+	"net"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
```

Comment:
if it is possible, we try not to introduce k8s dependencies.

Reply:
Not exactly. It depends on the value it brings and how many dependencies it involved.

But, usually, the `k8s.io/kubernetes` dependency should be restricted.

Reply:
got it.

Reply:
Because the flag check of k8s is relatively mature, it is considered to be introduced into karmada, which is a basic library and does not involve dependency on `k8s.io/kubernetes`